app/scripts/constants.js

```diff
@@ -28,6 +28,10 @@ window.OPENSHIFT_CONSTANTS = {
   CLI: {
     "Latest Release":          "https://github.com/openshift/origin/releases/latest"
   },
+  // Maps links to API UI URLs
+  API: {
+    "token-request": "https://api.preview.openshift.com/oauth/token/request"
```

Comment:
API URLs would never be in this file.  Anything associated with building URLs to the API should go through services/data.js